Deploy-CAM.ps1

```diff
@@ -3070,6 +3092,13 @@ function Set-VnetConfig() {
 
     # Now select subnets
     $vnet = Get-AzureRmVirtualNetwork -Name $vnetName -ResourceGroupName $vnetRgName
+    
+    # Only check if we are deploying to an existing CAM deployment, not to an existing domain controller.
+    # Domain controller version can use a different location, so we check that one later.
+    if(-not $deployOverDC){
+        Check-Location-Cores -rgLocation $vnet.Location -neededCores $minCoresAddConn
```

Comment:
How is $minCoresAddConn set here?

Reply:
$minCoresAddConn and $minCoresFullDeploy are both just hardcoded at 3 and 6 respectively. They will need to be updated manually with any changes to the default VMs.

Reply:
Updated to pass through parameters to the Set-VnetConfig function.